pkg/rbd/controllerserver.go

```diff
@@ -40,7 +41,45 @@ const (
 
 type controllerServer struct {
 	*csicommon.DefaultControllerServer
-	persistMetadata bool
+	MetadataStore util.CachePersister
+}
+
+var (
+	volPath = "controller"
+	snapPath = "controller-snap"
+
+	rbdVolumes = map[string]*rbdVolume{}
+	rbdSnapshots = map[string]*rbdSnapshot{}
+)
+
+func (cs *controllerServer) LoadExDataFromMetadataStore() {
+	if _, err := os.Stat(path.Join(PluginFolder, volPath)); os.IsNotExist(err) {
```

Comment:
`NodeCache` should make sure its local directories exist, let's keep it contained in one place.